src/Support/Contracts/FieldBuilderDirective.php

```diff
@@ -2,14 +2,18 @@
 
 namespace Nuwave\Lighthouse\Support\Contracts;
 
+use GraphQL\Type\Definition\ResolveInfo;
+
 interface FieldBuilderDirective extends Directive
 {
     /**
      * Add additional constraints to the builder.
      *
      * @param  \Illuminate\Database\Query\Builder|\Illuminate\Database\Eloquent\Builder  $builder  the builder used to resolve the field
+     * @param  array<string, mixed>  $args
+     * @param  \Nuwave\Lighthouse\Support\Contracts\GraphQLContext  $context
      *
      * @return \Illuminate\Database\Query\Builder|\Illuminate\Database\Eloquent\Builder the modified builder
      */
-    public function handleFieldBuilder(object $builder): object;
+    public function handleFieldBuilder(object $builder, $root, array $args, GraphQLContext $context, ResolveInfo $resolveInfo): object;
```

Comment:
This is a breaking change and should be listed as such in both CHANGELOG.md and UPGRADE.md

Reply:
I couldn't get changelog.md to get automatically generated, I tried a couple of tools like: 

But they give me the diff result like:

```diff
4a5,6
> use GraphQL\Type\Definition\ResolveInfo;
>
10a13,14
>      * @param  array<string, mixed>  $args
>      * @param  \Nuwave\Lighthouse\Support\Contracts\GraphQLContext  $context
14c18
<     public function handleFieldBuilder(object $builder): object;
---
>     public function handleFieldBuilder(object $builder, $root, array $args, GraphQLContext $context, ResolveInfo $resolveInfo): object;
```

That's not the format you have used which have + and -, would you mind telling me how to make one like yours?

Reply:
I hand-write it - it is quite easy to do, just annotate the codeblock as `diff` and put + or - in the beginning of lines.

Reply:
@spawnia Oh thank you for teaching me about this I thought you auto-generate it somehow!